services/api-server/src/simcore_service_api_server/services_rpc/wb_api_server.py

```diff
@@ -4,48 +4,178 @@
+from models_library.api_schemas_webserver.licensed_items import LicensedItemGetPage
+from models_library.licensed_items import LicensedItemID
+from models_library.resource_tracker_licensed_items_checkouts import (
+    LicensedItemCheckoutID,
+)
+from models_library.services_types import ServiceRunID
+from models_library.users import UserID
+from models_library.wallets import WalletID
 from servicelib.rabbitmq._client_rpc import RabbitMQRPCClient
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker.errors import (
+    CanNotCheckoutNotEnoughAvailableSeatsError,
+)
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker.errors import (
+    CanNotCheckoutServiceIsNotRunningError as _CanNotCheckoutServiceIsNotRunningError,
+)
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker.errors import (
+    LicensedItemCheckoutNotFoundError as _LicensedItemCheckoutNotFoundError,
+)
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker.errors import (
+    NotEnoughAvailableSeatsError,
+)
+from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
+    checkout_licensed_item_for_wallet as _checkout_licensed_item_for_wallet,
+)
+from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
+    get_available_licensed_items_for_wallet as _get_available_licensed_items_for_wallet,
+)
 from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
     get_licensed_items as _get_licensed_items,
 )
+from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
+    release_licensed_item_for_wallet as _release_licensed_item_for_wallet,
+)
 
+from ..exceptions.backend_errors import (
+    CanNotCheckoutServiceIsNotRunningError,
+    InsufficientNumberOfSeatsError,
+    LicensedItemCheckoutNotFoundError,
+)
 from ..exceptions.service_errors_utils import service_exception_mapper
 from ..models.pagination import PaginationParams
-from ..models.schemas.model_adapter import LicensedItemGet
+from ..models.schemas.model_adapter import LicensedItemCheckoutGet, LicensedItemGet
 
 _exception_mapper = partial(service_exception_mapper, service_name="WebApiServer")
 
 
+def _create_licensed_items_get_page(
+    *, licensed_items_page: LicensedItemGetPage, page_params: PaginationParams
+) -> Page[LicensedItemGet]:
+    page = create_page(
+        [
+            LicensedItemGet(
+                licensed_item_id=elm.licensed_item_id,
+                name=elm.name,
+                license_key=elm.license_key,
+                licensed_resource_type=elm.licensed_resource_type,
+                pricing_plan_id=elm.pricing_plan_id,
+                created_at=elm.created_at,
+                modified_at=elm.modified_at,
+            )
+            for elm in licensed_items_page.items
+        ],
+        total=licensed_items_page.total,
+        params=page_params,
+    )
+    return cast(Page[LicensedItemGet], page)
```

Comment:
MINOR: i think this should also work and is "perhaps more readable`"

```python
page: Page[LicencedItemGet] = create_page(...)
return page
```

Reply:
why not use an assert? what you propose @pcrespov is also some sort of casting.